exercises/SS22/ex_rnw/sol_Ex_1.Rnw

```diff
@@ -0,0 +1,202 @@
+%	
+\end{align*}
+%
+Similarly,
+%
+\begin{align*}
+%	
+	\E_{xy} \left[ \left(\frac1n \sum_{i=1}^n m(\yi)\right) \left(\frac1n \sum_{i=1}^n m(\yi)\right)   \right]
+%	
+	&= \frac{1}{n^2} \left(	\sum_{i=1}^n	\E_{xy} \left[   m(\yi)  \left( \sum_{i=1}^n m(\yi)\right)   \right]		\right) \\
+%	
+	&=  \frac{1}{n^2} \left(	\sum_{i=1}^n	\E_{xy} \left[   m(\yi)^2 + \sum_{j \neq i}  m(\yi) m(y^{(j)})   \right]		\right) \\
+	%	
+	&=  \frac{1}{n^2} \left(\sum_{i=1}^n	\E_{xy} \left[   m(\yi)^2  \right] +  	\sum_{i=1}^n \sum_{j \neq i} \E_{xy} \left[ m(\yi) m(y^{(j)})   \right]		\right) \\
+%	
+	&=  \frac{1}{n^2} \left( n	\E_{xy} \left[   m(y)^2  \right] +  	\sum_{i=1}^n \sum_{j \neq i} \E_{xy} \left[ m(\yi)  \right]	 \E_{xy} \left[ m(y^{(j)})   \right]		\right) \\
+%	
+	&=  \frac{1}{n^2} \left( n	\E_{xy} \left[   m(y)^2  \right] +  n(n-1) \E_{xy} \left[  m(y)   \right]^2		\right) \\
+	%	
+	&=  \frac1n	\E_{xy} \left[   m(y)^2  \right] +   (1-\frac1n) \E_{xy} \left[  m(y)   \right]^2	. \\
+%	
+\end{align*}
+
+%
+So, combining the three later math displays, we obtain
+%
+\begin{align*}
+	%
+	\risk_L\left(\fh \right) 
+	% 
+	&=   \E_{xy} \left[    m(y)^2  \right]  - 2 \E_{xy} \left[    m(y)   \frac1n \sum_{i=1}^n m(\yi)  \right]  +    \E_{xy} \left[ \frac1n \sum_{i=1}^n m(\yi)    \right]  \\
+	%
+	&=  \E_{xy} \left[    m(y)^2  \right]  - 2 \E_{xy} \left[  m(y) \right]^2 + \frac1n	\E_{xy} \left[   m(y)^2  \right] +   (1-\frac1n) \E_{xy} \left[  m(y)   \right]^2 \\
+%	
+	&= \left( 1 + \frac1n \right) \left(   \E_{xy} \left[    m(y)^2  \right]  -  \E_{xy} \left[    m(y)  \right] ^2			\right) \\
+%	
+	&= \left( 1 + \frac1n \right) \var(m(y)).
+	%
+\end{align*}
+% 
+\item In order to derive the risk minimizer, we consider the unrestricted hypothesis space $\Hspace = \{f: \Xspace \to \R\}$. 
+%
+	By the law of total expectation
+		\begin{eqnarray*}
+			\risk_L\left(f \right)  &=& \E_{xy} \left[\Lxy\right] 
+			\\ &=& \E_x \left[\E_{y|x}\left[\Lxy~|~\xv\right]\right] \\
+			&=& \E_x
+			\left[\E_{y|x}\left[(m(y)- m(\fx))^2~|\xv\right]\right]. 
+		\end{eqnarray*} 
+	%	
+	Since $\Hspace$ is unrestricted we can choose $f$ as we wish: At any point $\xv = \xv$ we can predict any value $c$ we want. The best point-wise prediction is 
+	$$
+	\fxbayes = \mbox{argmin}_c \E_{y|x}\left[( m(y) - m(c) )^2 ~|~ \xv  \right]\overset{(*)}{=} m^{-1} \left(  \E_{y|x}\left[ m(y) ~|~ \xv \right] \right),
+	$$
+%	
+	where $(*)$ is due to 
+%	
+	\begin{align*}
+%		
+		\mbox{argmin}_c \E\left[(m(y) - m(c))^2\right] 
+%		
+		&= \mbox{argmin}_c \underbrace{\E\left[( m(y) - m(c) )^2\right] - \left(\E[ m(y)] - m(c) \right)^2}_{= \var[ m(y) - m(c)] = \var[m(y)]} + \left(\E[m(y)] - m(c)\right)^2 \\
+%		
+		&=   \mbox{argmin}_c \var[m(y)] + \left(\E[m(y)] - m(c)\right)^2 
+%		
+		= m^{-1} \left( \E[m(y)]\right) . 
+%		
+	\end{align*}
+% 
+Note that we could have used the a similar substitution as in (a) here to derive $\fbayes.$
```

Comment:
Kann man hier am Ende noch eine Intuition dafür vermitteln, was dieses Ergebnis bedeutet? Warum das "spannend" ist? (if it is ;) )

Reply:
Ich habe dazu nochmal etwas geschrieben ;-)